xml/snmp_sysdescr.xml

```diff
@@ -5866,7 +5866,7 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="0" name="os.vendor" value="Sun"/>
     <param pos="0" name="os.family" value="Solaris"/>
     <param pos="0" name="os.product" value="Solaris"/>
-    <param pos="0" name="os.arch" value="sun4u"/>
+    <param pos="0" name="os.arch" value="Sparc"/>
```

Comment:
This looks reasonable. Nobody's 32-bit Sparc systems work anymore anyway, except for many a few classic computer collectors.